### PR TITLE
add n22261 logs

### PR DESCRIPTION
**Pull request by glozow**

### CODE REVIEW

File: _posts/2021-06-23-#22261.md [hunk 1]
```diff
@@ -83,9 +83,226 @@ commit:
...
+17:59 <LarryRuane> ok thanks (hadn't done that, will do)
+17:59 <michaelfolkson> duncandean is working on tests it appears
+17:59 <michaelfolkson> I think everyone agrees it should have tests
+17:59 ⇐ kexkey_ quit (~kexkey@static-198-54-132-116.cust.tzulo.com): Ping timeout: 252 seconds
+18:00 <glozow> shameless plug: if you're interested in same-txid-diff-wtxid stuff, i also recommend https://github.com/bitcoin/bitcoin/pull/22253
+18:00 <LarryRuane> often the tests are harder to write than the code being tested, I've noticed!
+18:00 <glozow> that's all we have time for :)
+18:00 <glozow> #endmeeting
+18:00 <glozow> This is a test: https://github.com/glozow/bitcoin/commit/5069a834ed06f2df5f43c416dc5a501c3b010b33
+
```
Comment (jonatack) [inline _posts/2021-06-23-#22261.md:307]: These newlines at the start and end are harmless but not needed.

File: _posts/2021-06-23-#22261.md [hunk 2]
```diff
@@ -83,9 +83,226 @@ commit:
...
+17:57 <glozow> right
+17:57 → kexkey joined (~kexkey@static-198-54-132-100.cust.tzulo.com)
+17:58 <glozow> no consensus failures. but worth fixing, ya?
+17:58 <lightlike> If i understand it correctly, "transactions not being relayed" would rather be a downside of the first fix: if we happen to have bad peers, broadcast to them, get a GETDATA but they don't relay any further, then we'd currently try again after ~24hours (when we have better peers), after the fix we wouldn't do this anymore.
+17:58 <LarryRuane> we're about out of time, so this would be too long of a discussion, but I notice there's no new or changed test code with this PR .. would be interesting to get into why, and generally, what kind of PRs need tests and what kind don't
+17:58 <michaelfolkson> LarryRuane: If you read the review comments tests were discussed
+17:59 <LarryRuane> ok thanks (hadn't done that, will do)
+17:59 <michaelfolkson> duncandean is working on tests it appears
+17:59 <michaelfolkson> I think everyone agrees it should have tests
+17:59 ⇐ kexkey_ quit (~kexkey@static-198-54-132-116.cust.tzulo.com): Ping timeout: 252 seconds
```
Comment (jonatack) [inline _posts/2021-06-23-#22261.md:301]: You can configure your IRC client to not display the joins and quits.  We didn't usually add these to the meeting logs.

File: _posts/2021-06-23-#22261.md [hunk 3]
```diff
@@ -83,9 +83,226 @@ commit:
...
+17:01 <lightlike> hi
+17:01 <glozow> did anyone get a chance to look at the notes or review the PR?
+17:02 <svav> Looked at the notes
+17:02 <michaelfolkson> Yup, maybe can't answer all the questions though
+17:03 <michaelfolkson> Took a while to get my head around why the fixes were needed
+17:03 <glozow> okie dokie
+17:03 → zakinator joined (~zakinator@c-73-152-14-173.hsd1.va.comcast.net)
+17:03 <glozow> so this PR is making changes to the `BroadcastTransaction()` function - can anyone tell me what this function does?
+17:03 → ru joined (~ru@122.174.144.66)
+17:03 <glozow> we're here: https://github.com/bitcoin/bitcoin/blob/7317e14a44c6efc545e6fb9bcedee7174e93a8fa/src/node/transaction.cpp#L29
```
Comment (jonatack) [inline _posts/2021-06-23-#22261.md:107]: The commit hashes can be shortened to 7 or 8 characters, if you think it's more readable in the meeting log.I often shortened them.

File: _posts/2021-06-23-#22261.md [hunk 4]
```diff
@@ -6,8 +6,8 @@ pr: 22261
 authors: [jnewbery]
 components: ["mempool", "p2p"]
 host: glozow
-status: upcoming
-commit:
+status: past
+commit: ba99f37
```
Comment (jonatack) [inline _posts/2021-06-23-#22261.md:10]: The tagged branch needs to be pushed in order for the generated link (here, to https://github.com/bitcoin-core-review-club/bitcoin/tree/pr22261) to work. See CONTRIBUTING.md.

**Review by jonatack — COMMENTED** [review]
Thanks @glozow, a few comments below.

**Comment by jonatack** [comment]
(You can test the final rendering locally, see the Development section of the README.)

**Comment by glozow** [comment]
Thanks @jonatack 

Removed the commit hash for now

File: _posts/2021-06-23-#22261.md (hunk 4 above)
Comment (jnewbery) [reply _posts/2021-06-23-#22261.md]: Done!

**Comment by jnewbery** [comment]
> Removed the commit hash for now, since the PR branch hasn't changed so there probably won't be confusion.

We add the commit in case the PR branch changes subsequent to the review club, so it's always possible to go back and inspect the exact branch that was being reviewed.

I've updated this commit to include the hash again, and pushed a tag to the pr review club bitcoin core branch.

Thanks for pushing these logs!

**Pull request merged**